pkgs/by-name/cl/clash-verge-rev/package.nix

```diff
@@ -1,19 +1,67 @@
-{ lib
-, clash-verge
-, fetchurl
+{
+  lib,
+  autoPatchelfHook,
+  clash-meta,
+  dpkg,
+  fetchurl,
+  libayatana-appindicator,
+  nix-update-script,
+  openssl,
+  stdenv,
+  udev,
+  webkitgtk,
+  wrapGAppsHook3,
 }:
 
-clash-verge.overrideAttrs (old: rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "clash-verge-rev";
-  version = "1.6.6";
+  version = "1.7.5";
 
   src = fetchurl {
-    url = "https://github.com/clash-verge-rev/clash-verge-rev/releases/download/v${version}/clash-verge_${version}_amd64.deb";
-    hash = "sha256-x+Xkasg6Yzft5CLg2YFCRkgpDeiVvvdmcLjrg+oIOT8=";
+    url = "https://github.com/clash-verge-rev/clash-verge-rev/releases/download/v${finalAttrs.version}/clash-verge_${finalAttrs.version}_amd64.deb";
+    hash = "sha256-pVEP+A4W6xLShFXuXPA6P+HZT8Hqkj/HRW2LaOOBI6U=";
   };
 
-  meta = old.meta // (with lib; {
+  nativeBuildInputs = [
+    dpkg
+    wrapGAppsHook3
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    openssl
+    webkitgtk
+    stdenv.cc.cc
```

Comment:
Isn't cc.cc included by default in the stdenv?

Reply:
I copied them from clash-verge. https://github.com/NixOS/nixpkgs/blob/aeda87464f545fbe4ebb26a79e547b097b04789a/pkgs/by-name/cl/clash-verge/package.nix#L33